src/cpp/include/openvino/genai/tokenizer.hpp

```diff
@@ -87,23 +87,59 @@ class OPENVINO_GENAI_EXPORTS Tokenizer {
     /**
     * @brief decode sequence of tokens
     * @param tokens vector storing tokens
+    * @param tokenization_params AnyMap with detokenization parameters, e.g. {"skip_special_tokens", false}
```

Comment:
```suggestion
    * @param detokenization_params AnyMap with detokenization parameters, e.g. ov::genai::skip_special_tokens(false)
```

Reply:
Applied the first part with `tokenization_params` -> `detokenization_params `. But the other half should remain, since it's method for AnyMap and it accepts map not `ov::Properties`, for `ov::Properties` we have a separate method.